Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java

```diff
@@ -0,0 +1,183 @@
+package edu.asu.conceptpower.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.conceptpower.core.ConceptType;
+
+@Component
+public class ConceptEditBean {
+
+	private Map<String, String> possMap = new LinkedHashMap<String, String>();
+	private String word;
+	private String concept;
+	private String selectedPosValue;
+	private String selectedListName;
+	private String conceptListValue;
+	private List conceptList;
+	private String description;
+	private String synonymsids;
+	private String conceptType;
+	private List conceptTypeList;
+	private String equals;
+	private String similar;
+	private String selectedTypeId;
+	private String selectedTypeName;
+	private ConceptType[] types;
+	private String conceptId;
+	private List conceptEntryList;
+
+	public String getConcept() {
+		return concept;
+	}
+
+	public void setConcept(String concept) {
+		this.concept = concept;
+	}
+
+	public String getConceptListValue() {
+		return conceptListValue;
+	}
+
+	public void setConceptListValue(String conceptListValue) {
+		this.conceptListValue = conceptListValue;
+	}
+
+	public String getDescription() {
+		return description;
+	}
+
+	public void setDescription(String description) {
+		this.description = description;
+	}
+
+	public String getConceptType() {
+		return conceptType;
+	}
+
+	public void setConceptType(String conceptType) {
+		this.conceptType = conceptType;
+	}
+
+	public List getConceptTypeList() {
+		return conceptTypeList;
+	}
+
+	public void setConceptTypeList(List conceptTypeList) {
+		this.conceptTypeList = conceptTypeList;
+	}
+
+	public String getEquals() {
+		return equals;
+	}
+
+	public void setEquals(String equals) {
+		this.equals = equals;
+	}
+
+	public String getSimilar() {
+		return similar;
+	}
+
+	public void setSimilar(String similar) {
+		this.similar = similar;
+	}
+
+	public String getSelectedPosValue() {
+		return selectedPosValue;
+	}
+
+	public void setSelectedPosValue(String selectedPosValue) {
+		this.selectedPosValue = selectedPosValue;
+	}
+
+	public Map<String, String> getPossMap() {
+		possMap.put("noun", "Nouns");
```

Comment:
also, check if there isn't an Enum that contains the values and/or labels. It's used in several places, so if there is non, there should be one.